macro-kroki-api/src/main/java/org/xwiki/contrib/kroki/configuration/KrokiMacroConfiguration.java

```diff
@@ -51,6 +51,11 @@
      */
     boolean isKrokiDockerContainerReusable();
 
+    /**
+     * @return {@code true} to use TLS if a host is specified by {@link #getKrokiHost()}
+     */
+    boolean getKrokiUseTls();
```

Comment:
```suggestion
    boolean getKrokiUseTLS();
```

Using upper case for acronyms is the preferred code style in XWiki.

Reply:
Gotcha I wasn't aware of that.